test/extended/util/disruption/controlplane/known_backends.go

```diff
@@ -56,6 +56,10 @@ func StartAllAPIMonitoring(ctx context.Context, recorder monitor.Recorder, clust
 		return err
 	}
 
+	if err := StartRemoteAPIMonitoring(ctx, recorder, clusterConfig, lb); err != nil {
```

Comment:
Absolutely not.  Fit into https://github.com/openshift/origin/pull/28077

Reply:
Done